spec/01-abstract.md

```diff
@@ -1,7 +1,39 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
+# Problem Statement

```

Comment:
Having H1 inside the abstract section looks bad in rendered document. "Abstract" is already and H1 title.

https://pr-preview.s3.amazonaws.com/danielkhan/distributed-tracing/pull/259.html#overview

![image](https://user-images.githubusercontent.com/9950081/54835707-e353fa80-4c7f-11e9-9a39-07450d19a3d0.png)